website/app.js

```diff
@@ -3,35 +3,65 @@ const apiEndPoint = "api.openweathermap.org";
 const apiKey = "081a059f88749d9d3c5a61e73985ef20";
 const country = "us";
 
-// Create a new date instance dynamically with JS
-let d = new Date();
-let newDate = d.getMonth() + "." + d.getDate() + "." + d.getFullYear();
```

Comment:
same: **const** declaration and also, **newDate** could be "todaysDate" or "todaysDateAsStr". More clear right?